platforms/Windows/platforms/windows/windows.wxs

```diff
@@ -867,6 +995,18 @@
       </ComponentGroup>
     <?endif?>
     <?if $(IncludeX86) = True?>
+      <ComponentGroup Id="_Builtin_float.x86" Directory="_Builtin_float.swiftmodule">
+        <Component DiskId="8">
+          <File Source="$(ExperimentalSDKRoot)\usr\lib\swift\windows\_Builtin_float.swiftmodule\i686-unknown-windows-msvc.swiftdoc" />
+        </Component>
+        <Component DiskId="8">
+          <File Source="$(ExperimentalSDKRoot)\usr\lib\swift\windows\_Builtin_float.swiftmodule\i686-unknown-windows-msvc.swiftmodule" />
+        </Component>
+
+        <Component Directory="WindowsExperimentalSDK_usr_lib_swift_windows_x86">
```

Comment:
missing disk assignment

Reply:
nevermind. we have it on the directory.

Reply:
Yeah, trying to minimise the locations where we have to specify the DiskId to avoid mistakes.